lib/providers/map_data.dart

```diff
@@ -1,17 +1,23 @@
+import 'dart:async';
+import 'package:concordia_navigation/models/itinerary.dart';
 import 'package:flutter/foundation.dart';
 import 'package:google_maps_flutter/google_maps_flutter.dart';
-import 'dart:async';
 import 'package:flutter/material.dart';
 
 ///Observer Pattern
 ///Handles all the data related to the map, listens to changes and notifies listeners.
 class MapData extends ChangeNotifier {
   Completer<GoogleMapController> _completer = Completer();
+  Itinerary itinerary;
 
   Completer<GoogleMapController> get getCompleter {
     return _completer;
   }
 
+  final double _expandedBottomSheetBottomPosition = 0;
+  final double _collapsedBottomSheetBottomPosition = -700;
```

Comment:
Perhaps we can use `SizeConfig` or move these contants to some other place?

Reply:
Oh yeah, completely forgot to use the `SizeConfig()` class